src/theme/material/assets/pager.target.scss

```diff
@@ -0,0 +1,12 @@
+.q-grid-pager-target-button {
+	mat-icon {
```

Comment:
use .q-grid-icon class,
and avoid to use .mat classes inside `material/assets` there is a special folder `material/assets/mat`